app/lib/widgets/polls/details/reply_card.dart

```diff
@@ -0,0 +1,226 @@
+import 'package:flutter/material.dart';
+import 'package:opengov_app/common.dart';
+import 'package:opengov_app/service/http_service.dart';
+import 'package:opengov_app/widgets/base/linked_text.dart';
+import 'package:opengov_app/widgets/polls/details/poll_details.dart';
+import 'package:opengov_app/widgets/polls/neapolitan.dart';
+import 'package:opengov_common/actions/feed.dart';
+import 'package:opengov_common/actions/vote.dart';
+import 'package:opengov_common/models/comment.dart';
+import 'package:opengov_common/models/reply.dart';
+import 'package:opengov_app/widgets/polls/reply_details.dart';
+import 'package:opengov_common/actions/vote_reply.dart';
+
+enum ReplyAction { agree, disagree, pass }
+
+extension ReplyActionScore on ReplyAction {
+  static const _scoreMap = {
+    ReplyAction.agree: 1,
+    ReplyAction.disagree: -1,
+    ReplyAction.pass: 0,
+  };
+
+  int get score => _scoreMap[this]!;
+}
+
+class ReplyCard extends StatefulWidget {
+  final ReplyBase reply;
+  final VoidCallback onActionPressed;
+
+  const ReplyCard({required this.reply, required this.onActionPressed});
+
+  @override
+  State<ReplyCard> createState() => _ReplyCardState();
+}
+
+class _ReplyCardState extends State<ReplyCard> {
+  static const _showReason = false;
+
+  final _reasonController = TextEditingController();
+
+  void _onHelpPressed() {
+    showMessageDialog(
+      context,
+      title: 'Share a reason',
+      body: 'You can optionally include a reason along with your vote. Your '
+      'reason will only be visible to Crimson OpenGov admins, who will '
+      'share the reasons with Harvard administrators making decisions.\n\n'
+      'However, just like your votes and responses, all reasons are '
+      'anonymous–no personally identifying info is shared with your '
+      'responses.',
+    );
+  }
+
+  Future<void> _onVotePressed(ReplyAction action) async {
+    final reason = _reasonController.text;
+    final response = await HttpService.voteReply(VoteReplyRequest(
+        replyId: widget.reply.id, score: action.score, reason: reason));
+
+    if (response?.success ?? false) {
+      widget.onActionPressed();
+    }
+  }
+
+  /*
```

Comment:
There's a bunch of commented-out code throughout these files. Can you delete all of it?